### DISCUSSION TITLE
To do a manual install, where can I find main.js?

### DISCUSSION THREAD

**Discussion by lennardong** [vslinko/obsidian-zoom#47] [Q&A] [answered]
Hi, 

To do a manual install, where can I find `main.js`? 

I'm trying to do a manual install an version of zoom to work with obsidian legacy edit mode. I'm stuck in where to find the main.js to be included into the vault folder. 

Any help on where to find main.js appreciated! 

-lenn

**Comment by vslinko** [comment, marked as answer]
@lennardong hi! you can find assets in releases section https://github.com/vslinko/obsidian-zoom/releases

**Comment by lennardong** [comment]
Thanks Vslinko! Learned something new. :)

*(I was looking "under tags")*

cheers,
Lenn

On Sun, 26 Dec 2021 at 17:50, Viacheslav Slinko ***@***.***>
wrote:

> @lennardong <https://github.com/lennardong> hi! you can find assets in
> releases section https://github.com/vslinko/obsidian-zoom/releases
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vslinko/obsidian-zoom/discussions/47#discussioncomment-1871389>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHDGGILEWGLDQWWQ4HTBPMTUS3QPDANCNFSM5KYUGLKQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

-------------------------------

-------------------------------
